straxen/plugins/pulse_processing.py

```diff
@@ -125,6 +125,7 @@ class PulseProcessing(strax.Plugin):
 
     provides = ('records', 'veto_regions', 'pulse_counts')
     data_kind = {k: k for k in provides}
+    save_when = strax.SaveWhen.TARGET
```

Comment:
This is a very drastic change and will disable record matrix et cetera plotting because you cannot do time selections on data that is not stored!